requirements-test.txt

```diff
@@ -10,3 +10,4 @@ coveralls==1.9.2
 pytest-cov==2.8.1
 py-mini-racer==0.1.18
 statsmodels==0.10.2
+git+git://github.com/scikit-learn-contrib/lightning.git@b96f9c674968496e854078163c8814049a7b9f43
```

Comment:
0.5.0 from PyPI fails with Python 3.7.
Refer to https://github.com/scikit-learn-contrib/lightning/pull/133, https://github.com/scikit-learn-contrib/lightning/issues/139, https://github.com/scikit-learn-contrib/lightning/issues/132.